test/Compilation/Positive.hs

```diff
@@ -455,5 +455,10 @@ tests =
         "Test076: Builtin Maybe"
         $(mkRelDir ".")
         $(mkRelFile "test076.juvix")
-        $(mkRelFile "out/test076.out")
+        $(mkRelFile "out/test076.out"),
+      posTestEval
```

Comment:
Can this be `posTest` instead of `posTestEval` so we test compilation post-core?

Reply:
my understanding is that after core it doesn't make a difference wether a field is an instance field or not. Should I still change it to `posTest`?

Reply:
We can keep it like this if you like. The reason to change it is to add more test coverage to the post-core pipeline.

Reply:
I've made the change. I think it is an overall relevant test.